templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -317,33 +322,28 @@ contents:
           echo "Static IP addressing detected on default gateway connection: ${old_conn}"
           # clone the old connection to get the address settings
           # prefer cloning vs copying the connection file to avoid problems with selinux
-          nmcli conn clone "${old_conn}" "${ovs_interface}"
+          clone_nm_conn "${old_conn}" "${ovs_interface}"
           shopt -s nullglob
-          new_conn_files=(${NM_CONN_CONF_PATH}/"${ovs_interface}"*)
+          new_conn_files=(${NM_CONN_RUN_PATH}/"${ovs_interface}"*)
```

Comment:
quotes `"${NM_CONN_RUN_PATH}"` ?

Reply:
constant with no spaces so there shouldn't be a problem for now  I guess